infrastructure/networked_rds/README.md

```diff
@@ -29,11 +29,11 @@ module "networked_rds" {
 
   project_name = "foodoasis"
   stage = "dev"
-  region = "us-west-2"
+  region = "us-east-2"
```

Comment:
I believe food oasis is running in us-west-2 (Oregon) region to be closer proximity to main traffic source in CA. 
Why choose this region?

Reply:
I see they have a dev db on east-2. I wanted to test my code so I though for dev enviornment, I should choose the same region

Reply:
I understand, yes that makes sense